### ISSUE TITLE
No support for recurrence (RRULE)

### ISSUE THREAD

**Issue by drzraf** [jrvarma/icalcli#6]
```
BEGIN:VEVENT
UID:1
SUMMARY:AA
DTSTART:20220510
DURATION:P5D
RRULE:FREQ=DAILY;INTERVAL=10
COLOR:red
END:VEVENT
```

**Actual**
```
┌──────────┬──────────┬──────────┬──────────┬──────────┬──────────┬──────────┐
│dimanche  │lundi     │mardi     │mercredi  │jeudi     │vendredi  │samedi    │
├──────────┼──────────┼──────────┼──────────┼──────────┼──────────┼──────────┤
│08 mai    │09 mai    │10 mai    │11 mai    │12 mai    │13 mai ** │14 mai    │
│          │          │          │          │          │          │          │
│          │          │AA        │AA        │AA        │AA        │AA        │
│          │          │          │          │          │          │          │
├──────────┼──────────┼──────────┼──────────┼──────────┼──────────┼──────────┤
│15 mai    │16 mai    │17 mai    │18 mai    │19 mai    │20 mai    │21 mai    │
│          │          │          │          │          │          │          │
├──────────┼──────────┼──────────┼──────────┼──────────┼──────────┼──────────┤
│22 mai    │23 mai    │24 mai    │25 mai    │26 mai    │27 mai    │28 mai    │
│          │          │          │          │          │          │          │
├──────────┼──────────┼──────────┼──────────┼──────────┼──────────┼──────────┤
│29 mai    │30 mai    │31 mai    │01 juin   │02 juin   │03 juin   │04 juin   │
│          │          │          │          │          │          │          │
├──────────┼──────────┼──────────┼──────────┼──────────┼──────────┼──────────┤
│05 juin   │06 juin   │07 juin   │08 juin   │09 juin   │10 juin   │11 juin   │
│          │          │          │          │          │          │          │
└──────────┴──────────┴──────────┴──────────┴──────────┴──────────┴──────────┘
```

**Expected**: Recurrence (restarting 20th)

**Comment by jrvarma** [comment]
Recurrence rules are not currently implemented at all. That is on my TODO list, but implementing RRULE is quite complex and unless I can find some open source code that I can reuse easily, this is not likely to happen in a hurry.

**Comment by jrvarma** [comment]
Yes, this looks doable now. When I have time, I will work on this.

**Comment by jrvarma** [comment]
This has now been implemented in version 1.0.0 which is now on `github` and `pypi`
